src/components/PostsItem.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="posts-item">
+    <h1>{{ openedPost.title }}</h1>
```

Comment:
Isn't it good to split Post component to wrapper, body (3 types) and comments list?